overlays/low-resource/kustomization.yaml

```diff
@@ -103,13 +103,13 @@ patches:
     name: codeintel-db
     group: apps
     version: v1
-  path: delete-resources-2.yaml    
+  path: delete-resources-4.yaml    
 - target:
     kind: Deployment
     name: codeinsights-db
     group: apps
     version: v1
-  path: delete-resources-2.yaml  
+  path: delete-resources-3.yaml  
```

Comment:
We'll bump this to 4 when https://github.com/sourcegraph/deploy-sourcegraph/pull/4111 is merged